public/app/features/explore/Logs/LogsTableDataplane.test.tsx

```diff
@@ -0,0 +1,171 @@
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
I would move this into a `LogsTable.test.tsx` as it is the same test as with Loki/Elastic format, just with data plane format. I don't understand why would it be in another file.